packages/internal-lib-build/bin/internal-lib-build.js

```diff
@@ -17,16 +17,14 @@ const execSync = (cmd, opts) => {
     return _execSync(cmd, {...defaults, ...opts})
 }
 
-const pkgRoot = p.join(__dirname, '..')
-
 const binDir = p.join(require.resolve('@babel/cli'), '..', '..', '..', '.bin')
 
 const main = () => {
     program.description(`Internal build tools for monorepo libraries`)
 
     program.command('build').action(() => {
         const babel = p.join(binDir, 'babel')
-        const babelConfig = p.resolve(p.join(process.cwd(), 'babel.config.js'))
+        const babelConfig = p.resolve(p.join(process.cwd(), '..', '..', 'babel.config.js'))
         const prepareDist = p.resolve(p.join(__dirname, 'prepare-dist.js'))
```

Comment:
```suggestion
        const babelConfig = p.join(__dirname, '..', '..', 'babel.config.js')
        const prepareDist = p.join(__dirname, 'prepare-dist.js')
```
If you're looking for a specific config in a specific place, you probably shouldn't use `process.cwd()`. (Also `resolve` is unnecessary here.)